site.conf

```diff
@@ -31,7 +31,7 @@
 
     -- List of NTP servers in your community.
     -- Must be reachable using IPv6!
-    ntp_servers = {'ntp.ffrn.de'},
+    ntp_servers = {'ntp.ffrn.de', '0.freifunk.pool.ntp.org', 'de.pool.ntp.org'},
```

Comment:
```suggestion
    ntp_servers = {'ntp.ffrn.de', '2.freifunk.pool.ntp.org', '2.de.pool.ntp.org'},
```

Reply:
According to https://linuxreviews.org/IPv6-listening_NTP_servers, the NTP pools starting with 2 point to IPv6-ready servers. Both updated names provide indeed IPv6 addresses in addition it IPv4 addresses.